projects/dsp-ui/src/lib/viewer/services/value-operation-event.service.ts

```diff
@@ -17,29 +17,38 @@ export class ValueOperationEventService {
 
     // Used in the listening component.
     // i.e. this.valueOperationEventSubscription = this._valueOperationEventService.on(Events.ValueAdded, () => doSomething());
-    on(event: Events, action: (newValue) => void): Subscription {
+    on(event: Events, action: (newValue: any) => void): Subscription {
```

Comment:
Instead of `any`, could you use a more restrictive type annotation?

Reply:
done in fd17180